python/doc/examples/probabilistic_modeling/random_vectors/plot_random_vector_manipulation.py

```diff
@@ -6,15 +6,22 @@
 # %%
 # The :class:`~openturns.RandomVector` object represents the concept of random variable.
 #
-# In this example we are going to exhibit some of its main methods.
+# This class can be cerated by:
+#
+# - Case 1: directly using the class :class:`~openturns.RandomVector`,
+# - Case 2: from python, using the class :class:`~openturns.PythonRandomVector` which enables to
+# redefine some methods (as the sampling method for example).
```

Comment:
indent is not right: "redefine" must be aligned with "Case 2"

Reply:
thanks......